README.md

```diff
@@ -3,7 +3,7 @@
 # Leaderboard
 In this project I will set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules. In the first stages I will be working on the version of the app following a wireframe, but without styling - just focus on functionality. In following activities, I will consume the Leaderboard API using JavaScript async and await and add some styling.
 
-![screenshot](./milestone2.png)
+![screenshot](./final-style.png)
```

Comment:
- [ ] Kindly deploy the project online and provide the right link on the readme.